Analysis/EventFiltering/filterTables.h

```diff
@@ -28,19 +28,54 @@ DECLARE_SOA_COLUMN(DG, hasDG, bool); //! Double Gap events, DG
 
 } // namespace filtering
 
-DECLARE_SOA_TABLE(NucleiFilters, "AOD", "Nuclei Filters", //!
+// nuclei
+DECLARE_SOA_TABLE(NucleiFilters, "AOD", "NucleiFilters", //!
```

Comment:
Please put comments everywhere in the next PR

Reply:
👍